src/main/java/core/basesyntax/FileWork.java

```
@@ -8,7 +13,23 @@
  * Результат: web wide width world</p>
  */
 public class FileWork {
+    private static final String PATTERN = "w";
+
     public String[] readFromFile(String fileName) {
-        return null;
+        StringBuilder sb = new StringBuilder();
+        try {
+            String string = Files.readString(Paths.get(fileName));
+            String[] stringArray = string.toLowerCase().split(" ");
+            for (String s : stringArray) {
+                if (s.startsWith(PATTERN)) {
+                    sb.append(s.replaceAll("\\W", "")).append(" ");
+                }
+            }
+        } catch (IOException e) {
+            e.printStackTrace();
```

do not ignore exception, re-throw it